core/template.js

```diff
@@ -13,10 +13,18 @@
 		rePlaceholder = /{([^}]+)}/g,
 		reEscapableChars = /([\\'])/g,
 		reNewLine = /\n/g,
-		reCarriageReturn = /\r/g;
+		reCarriageReturn = /\r/g,
+		isIndirectEvalSupported = true;
+
+	try {
+		Function( '1' )();
```

Comment:
@rpicard this is a one time operation for detection. as far as I can tell, there is no other way to detect CSP `unsafe-eval` rule. Ideally, we settle on the idea to use only the regex solution going forward, instead of a forked logic that requires detection.